discovery.go

```diff
@@ -25,6 +25,36 @@ const (
 	DiscoveryModeOnce
 )
 
+type BucketsSearchMode int
+
+const (
+	// BucketsSearchLegacy implements the same logic as lua router:
+	// send bucket_stat request to every replicaset,
+	// return a response immediately if any of them succeed.
+	BucketsSearchLegacy BucketsSearchMode = iota
+	// BucketsSearchBatchedQuick and BucketsSearchBatchedFull implement another logic:
+	// send buckets_discovery request to every replicaset with from=bucketID,
+	// seek our bucketID in their responses.
+	// Additionally, store other bucketIDs in the route map.
+	// BucketsSearchBatchedQuick stops iterating over replicasets responses as soon as our bucketID is found.
+	// BucketsSearchBatchedFull always iterates over all replicasets responses.
+	BucketsSearchBatchedQuick
+	BucketsSearchBatchedFull
+)
+
+func (mode BucketsSearchMode) String() string {
+	switch mode {
+	case BucketsSearchLegacy:
+		return "BucketsSearchLegacy"
```

Comment:
for such cases lets use //go:generate stringer -type BucketsSearchMode -linecomment
intead

Reply:
Fixed: I removed this code and don't log anymore it's value